app/assets/javascripts/initializers/initializeFetchFollowedArticles.js.erb

```diff
@@ -1,10 +1,3 @@
-function initializeFetchFollowedArticles() {
-  if ( document.getElementById("featured-story-marker") && checkUserLoggedIn() ) {
```

Comment:
`document.getElementById("featured-story-marker")` is always null because this gets loaded via Preact components when logged on so `algoliaFollowedArticles()` is never called.

`insertTimes()` does nothing because the HTMLCollection is and empty NodeList.